tests/rbd/rbd_utils.py

```diff
@@ -396,6 +397,21 @@ def list_images(self, pool_name):
         images = json.loads(out)
         return images
 
+    def image_status(self, image, **kw):
+        """Get Image status.
+
+        Command: rbd status <pool-name>/<image-name>
+
+        Args:
+            image: name of the image (ex., pool1/image1)
+            kw: other test parameters ( ex., {output: True, json: true})
```

Comment:
[Optional]

Change image to image specification / image spec
image could be confused with actual image name, mentioning as optional as example will help to understand
Usually `pool/image` is called image specification

Reply:
(Duplicate comment)